deployment/k8s/charts/aggregates/testClusters/environments/es-5-6-single-node-cluster.yaml

```diff
@@ -0,0 +1,31 @@
+conditionalPackageInstalls:
+  proxy: false
+  source: true
+  target: false
+
+source:
```

Comment:
I think the way this is set up is that this is _always_ a source cluster when it's included with the aggregate test cluster chart. Is that true? What kind of an error is thrown if you specify two source clusters? I'm fine with this being only a source, but we should be clear about it (and it's a bigger deal for those with overlap, like 7.10 or OS 1, which could be a source or a target--how are those handled?)

Reply:
Not a blocker for now, but worth having a note/chart on what combos currently work, and maybe a JIRA task about what has to be done to accommodate it.